src/Server/Provision/ServerProvisioner.php

```diff
@@ -393,8 +471,6 @@ private function getRaidLevel()
      */
     private function addons(array $choices)
     {
-        $addons = $this->multiChoice($choices, '/Add On ([0-9]+)/');
```

Comment:
why was this removed? Looks like it's needed below?

Reply:
I moved it outside of the function because I need to call the function and dont want that run on my code, there's a very similar function for disks and its called outside of that one but not this one.

Reply:
its a private function and is only called once in the code so i figure its nbd

Reply:
My point is, on the next line, there's an `array_filter($addons)`. That `$addons` variable no longer exists since it was removed here.